.devcontainer/Dockerfile

```diff
@@ -2,5 +2,14 @@ FROM ghcr.io/containerbase/node:18.16.0@sha256:fc48ac97eb5d0dee207d4a336a52d7d3b
 
 USER root
 
+RUN ls
+
```

Comment:
```suggestion
```

Reply:
@viceice I am still working on the PR. It is in draft state. I've used the ls command to help me in debugging.